styles.css

```diff
@@ -0,0 +1,466 @@
+@import url("https://fonts.googleapis.com/css2?family=Play&display=swap");
+body {
+  font-family: "Play", sans-serif;
+  background-color: black;
+}
+
+.animate__animated.animate__fadeIn {
+  --animate-duration: 1s;
+}
+
+.gameTitle {
+  font-size: 100px;
+  text-align: center;
+}
+
+.overallScreen {
+  background-color: #000000;
+  border-radius: 10px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  height: 800px;
+  width: 1000px;
+}
+
+.combinationsDisplay {
+  border-radius: 10px;
+  flex: 0.06;
+  display: flex;
+  flex-direction: row;
+  align-items: space-around;
+  font-size: 32px;
+  margin: 10px;
+  position: relative;
+  overflow: hidden;
+}
+
+.nameOfCombiDisplay {
+  list-style-type: none;
+  background-color: rgb(31, 31, 31);
+  color: white;
+  flex: 4;
+  display: flex;
+  flex-direction: column;
+  font-size: 32px;
+}
+
+.pCol1 {
+  flex: 1;
+  background-color: black;
+  padding-top: 1px;
+  color: #fff;
+  text-shadow: 0px 0px 10px #c9cece, 0px 0px 20px #c9cece, 0px 0px 30px #c9cece,
+    0px 0px 40px #c9cece;
+}
+
+.pCol2 {
+  flex: 1;
+  background-color: black;
+  padding-top: 2px;
+  color: #fff;
+  text-shadow: 0px 0px 10px #c7c345, 0px 0px 20px #c7c345, 0px 0px 30px #c7c345,
+    0px 0px 40px #c7c345;
+}
+
+.pCol3 {
+  flex: 1;
+  background-color: black;
+  padding-top: 2px;
+  color: #fff;
+  text-shadow: 0px 0px 10px #46bbbb, 0px 0px 20px #46bbbb, 0px 0px 30px #46bbbb,
+    0px 0px 40px #46bbbb;
+}
+
+.pCol4 {
+  flex: 1;
+  background-color: black;
+  padding-top: 2px;
+  color: #fff;
+  text-shadow: 0px 0px 10px #f95dab, 0px 0px 20px #f95dab, 0px 0px 30px #f95dab,
+    0px 0px 40px #f95dab;
+}
+
+.pCol5 {
+  flex: 1;
+  background-color: black;
+  padding-top: 2px;
+  color: #fff;
+  text-shadow: 0px 0px 10px #f38a0a, 0px 0px 20px #f38a0a, 0px 0px 30px #f38a0a,
+    0px 0px 40px #f38a0a;
+}
+
+.pOutBox {
+  padding: 4px;
+  text-align: center;
+  /* border-style: dashed; */
+}
+
+.cardsContainer {
+  background-color: black;
+  position: relative;
+  flex: 0.6;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.cover-card {
+  height: auto;
+  width: 17%;
+  border-radius: 15px;
+}
+
+.gameOver {
+  position: absolute;
+  width: 100%;
+  top: 30px;
+  left: 414px;
+  font-size: 30px;
+  color: white;
+}
+
+.card {
+  vertical-align: top;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex: 1;
+  height: 55%;
+  width: auto;
+}
+
+.holdStatus {
+  align-items: center;
+  text-align: center;
+  flex: 1;
+  margin-top: -54px;
+  font-size: 30px;
+  color: white;
+}
+
+.cardImage {
+  vertical-align: top;
+  flex: 5;
+  height: 100%;
+  width: auto;
+}
+
+.statsDisplay {
+  background-color: black;
+  flex: 0.2;
+}
+
+.buttonsContainer {
+  /* background-color: grey; */
```

Comment:
Please remove commented-out code before submitting, or if not, add a comment to explain why code may be commented out